tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -487,98 +487,93 @@ export const head_acc_emissive: Feature<boolean[]> = {
   component: FeatureTriBoolInput,
 };
 
-export const ipc_screen_toggle: FeatureToggle = {
-  name: 'IPC Screen',
-  description:
-    "Add some lore for your species! Won't show up if there's no custom species.",
-  component: CheckboxInput,
-};
-
 export const feature_ipc_screen: Feature<string> = {
   name: 'IPC Screen Selection',
   description:
     'Want to have a fancy species name? Put it here, or leave it blank.',
   component: FeatureDropdownInput,
 };
 
-export const ipc_screen_color: Feature<string[]> = {
-  name: 'IPC Screen Colors',
+export const ipc_screen_color: Feature<string> = {
+  name: 'IPC Screen Color',
   description:
     'Want to have a fancy species name? Put it here, or leave it blank.',
```

Comment:
Should probably change all of these descriptions.

Reply:
None of them ever show, but I'll update them so devs know what they do, I suppose.